GP/kernel_modules/kernel_utils.py

```diff
@@ -0,0 +1,14 @@
+# Author: Henry Moss & Ryan-Rhys Griffiths
```

Comment:
You can change the author name to yours!

Reply:
Haha was waiting for permission